### ISSUE TITLE
psmeca dump option

### ISSUE THREAD

**Issue by 79seismo** [GenericMappingTools/gmt#80]
When you are using psmeca with -Sa flag (Aki and Richards convention where you specify only one fault plane), write out results of the auxiliary plane and P and T axes. These are already plotted but there is no option for outputing them.

**Comment by seisman** [comment]
Although it's straightforward to output focal mechanism information, I think it's better to keep it as a simple plotting tool rather than a focal mechanism calculator.

Feel free to open it if someone still would like to add such feature to psmeca.

**Comment by 79seismo** [comment]
Not sure why it's hard to output the other nodal plane because it's already being calculated within GMT.

**Comment by seisman** [comment]
No, it's not hard. The problem is, nodal plane information is just a subset of full focal mechanism. If we output nodal plane, why not also output T, N, P axis information, full moment tensor or some other related information. 

BTW, the mechanism codes in GMT is now buggy and not fully tested.

**Comment by 79seismo** [comment]
I've used my own code for calculations, and so far, GMT output has been consistent with that.

**Comment by joa-quim** [comment]
I see output P,T,N as an interesting feature. But it should be made with the IO mechanism to allow sending to external interfaces, not simple fprintf()
